src/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -20,10 +21,11 @@ const AuctionsApp: React.FC<AuctionsAppProps> = props => {
   // FIXME: Remove once new filter launches
   const enableNewAuctionsFilter = getENV("ENABLE_NEW_AUCTIONS_FILTER")
 
+  const hasMyBids = viewer.me?.myBids?.active?.length
```

Comment:
We should do: 

```tsx
const hasMyBids = !!viewer.me?.myBids?.active?.length
```
or 
```tsx
const hasMyBids = Boolean(viewer.me?.myBids?.active?.length)
```
or 
```tsx
const hasMyBids = viewer.me?.myBids?.active?.length > 0
```

The reason we need to coerce it into a boolean is because in JSX, something like 

```tsx
const foo = 0 

<div>{foo && "hi"}</div>
```
will actually emit 
```tsx
<div>0</div>
```
in the html. Numbers are totally valid outputs, while booleans will safely guard.

Reply:
Yes, this was really annoying, will update!